src/components/Navbar/Navbar.tsx

```diff
@@ -1,18 +1,21 @@
 import React from 'react';
 import styles from './Navbar.module.css';
+import Navbar from 'react-bootstrap/Navbar';
 import web from 'assets/talawa-logo-lite-200x200.png';
```

Comment:
Can you change this from `web` to `TalawaLogo` or `Logo`?

I missed it in my last review
```suggestion
import Logo from 'assets/talawa-logo-lite-200x200.png';
```

Reply:
@DangaRanga Noted